Detailed-README.md

```diff
@@ -156,6 +156,10 @@ split_client.get_treatment('user_id','feature_name', attr: 'val')
 
 *default value* = `false`
 
+**ip** : Machine IP (used in Redis namespace and sent to the Split API)
```

Comment:
This is not right. We already have `@machine_ip = SplitConfig.get_ip` in split_config.

We need the consumer to use it while sending the impression to the cache abstraction. When redis is used, in this case, we'll use it to decide on the key path.